docs/source/release-history.rst

```diff
@@ -2,6 +2,11 @@
 Release History
 ***************
 
+v0.1.3 (2021-03-29)
+===================
+* added log propagation configuration to reduce log noise
```

Comment:
https://github.com/NSLS-II/nslsii/pull/119 seems to be of a different purpose. I think the log propagation configuration was covered earlier (https://github.com/NSLS-II/nslsii/pull/112), and was missing from the release notes for v0.1.2. Maybe it should be moved there, and this bullet point to be covered by the summary of #119?